pkg/cmd/cli/uninstall/uninstall.go

```diff
@@ -89,12 +87,13 @@ Use '--force' to skip the prompt confirming if you want to uninstall Velero.
 }
 
 // Run removes all components that were deployed using the Velero install command
-func Run(ctx context.Context, client *kubernetes.Clientset, extensionsClient *apiextensionsclientset.Clientset, namespace string, waitToTerminate bool) error {
+func Run(kbClient kbclient.Client, namespace string, waitToTerminate bool) error {
```

Comment:
I believe we should still pass the `context.Context` in, because cancellation of contexts cascades.

By hard-coding the background context (L136 and 153 in this file are examples), that cascading effect is lost and the only cancellation that can happen in this function is the background one.

I realize that we passed the background context in at the call site originally, but it's possible that some other user of the function would find need for using a different context.